app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }, allow_nil: true
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }, allow_nil: true
+
```

Comment:
- [ ] The `comments_counter` and `likes_counter` columns should be of data type integer. This means that they should not have `nil` values because `nil` is another data type. Also, setting validations to disallow nil values for a column ensures data integrity and helps prevent potential errors or unexpected behaviour. Kindly remove the validation that allows the `posts_counter` column to store `nil` values. 👍 😎